.gitignore

```diff
@@ -0,0 +1,53 @@
+HELP.md
+.gradle
+build/
+local-dev/terra-helm/
+local-dev/terra-helmfile/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**
+!**/src/test/**
+
+### STS ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+out/
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+
+### VS Code ###
+.vscode/
+
+### Emacs backup files
+*.*~
+
+### Mac directory metadata
+.DS_Store
+
+### OpenAPI CodeGen detritus
```

Comment:
This is a bit suspect, in particular globally ignoring pom.xml. There's a `.openapi-generator-ignore` file for the purpose of omitting certain generated artifacts, feels like that could be used here instead - or if it's broken for some reason, maybe this could be more targeted to a specific directory